main.cpp

```diff
@@ -283,35 +445,98 @@ struct CarWash //2)
     2)store data read from sensors
     3)store status 
  */
-
+struct Memory
+{
+    //1)size
+    int memorySize = 1000;
+    //2)type of memory (ram , eprom, rom, flash)
+    char memoryType = 'r';
+    //3)clock speed
+    int memorySpeed = 1000;
+    //4)slot for flash card
+    char memoryFlash = 'y';
+    //5)power consumption
+    int powerComsumption = 200;
+
+    //1)store script
+    int writeScript( int scriptSize = 500 );
+    //2)store data read from sensors
+    int writeData( int memoryLocation = 800, int dataSize = 1 );
+    //3)store status
+    int writeStatus( int statusCode = 0 );
+};
 /*
 9)reset button
 5 properties:
     1)size
     2)location on the board
     3)color
     4)material
-    5)useful life
+    5)number of clicks before failure
 3 things it can do:
     1)reset the script
     2)produce a click sound
     3)trigger a led
  */
-
+struct ResetButton
+{
+    //1)size
+    float buttonHeight = 0.05f;
+    //2)location on the board
+    char buttonLocation = 't';
+    //3)color
+    char buttonColor = 'r';
+    //4)material
+    char buttonMaterial = 'p';
+    //5)number of clicks before failure
+    int buttonClicksBeforeFailure = 10000;
```

Comment:
using 'button' in the member variable names of a class named `Button` is redundant
- [ ] revise

Reply:
ok, removed word 'button' from member variables.